src/app/shared/components/declarative-form/declarative-form.component.html

```diff
@@ -354,7 +354,7 @@
           <ng-template [ngSwitchCase]="'container'">
             <div class="row">
               <div *ngFor="let innerField of field.fields"
-                   [fxFlex]="innerField.flex ? innerField.flex : ''">
+                   [style.width.%]="innerField.flex ? innerField.flex : ''">
```

Comment:
@votdev I cannot find any solution with Bootstrap because `w-` class accept only values `0,25,50,75,100`. Do you have any tips?

Reply:
In that case we need to adapt the field configuration to what Bootstrap provides.
Maybe we can change
```
flex?: number;
```
to
```
flex?: 25 | 50 | 75 | 100;
```

Reply:
I think so.

Reply:
What about https://getbootstrap.com/docs/5.0/layout/grid/#setting-one-column-width? In that case we have 12 colums.